examples/datasets/front_3d_object_sampling/main.py

```diff
@@ -0,0 +1,150 @@
+import blenderproc as bproc
+import os
+import numpy as np
+import argparse
+#import bpy
+
+from blenderproc.python.utility.CollisionUtility import CollisionUtility
```

Comment:
That should also be added to the public API if it is used in an example.

Reply:
I removed it as it is not used anywhere